src/packages/migrate/src/MigrateEngine.ts

```diff
@@ -224,11 +230,7 @@ export class MigrateEngine {
           if (code !== 0 || messages.includes('panicking')) {
             let errorMessage =
               chalk.red.bold('Error in migration engine: ') + messages
-            if (code === 250) {
-              // Not a panic
-              // It's a UserFacingError https://github.com/prisma/prisma-engines/pull/1446
-              errorMessage = chalk.red.bold('UserFacingError')
-            } else if (this.lastError && code === 255) {
+            if (this.lastError && code === 101) {
```

Comment:
It would be nice to have the Error Codes as an enum/object so it makes this easier to read. Something like `ErrorCode.UserFacingError`

Reply:
Good point!